edx_solutions_projects/utils.py

```diff
@@ -0,0 +1,30 @@
+import boto3
+
+from django.conf import settings
+
+
+S3_FILE_URL_TIMEOUT = 60 * 30  # 30 minutes
```

Comment:
Would be better if we could pick up from settings, or something like that.
`settings.get('S3_FILE_URL_TIMEOUT', DEFAULT_S3_FILE_URL_TIMEOUT)`

Reply:
Actually this is very specific to GW submission files that's why we didn't put it in settings